doc/design/OrionMongoSink.md

```diff
@@ -118,3 +175,5 @@ Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) **[Contributor and
 Germán Toro del Valle (german.torodelvalle@telefonica.com) **[Contributor]**
 <br>
 Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
+
+[Top](#top)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/51d342c1565cf1b6efa79da68154b241c77cb4b7